### PR TITLE
Create search component

### PR DESCRIPTION
**Pull request by Brolyus**
Added the search menu, plus we merged our branches and edited our settings with prettier to have the same conf on all computers

### CODE REVIEW

File: front/src/App.js [hunk 1]
```diff
@@ -28,15 +31,47 @@ class App extends React.Component {
...
+    handleSelectedTags(instrument) {
+        this.state.selectedTags.includes(instrument)
+            ? this.setState({
+                  selectedTags: this.state.selectedTags.filter(
+                      tag => tag !== instrument
+                  )
+              })
+            : this.setState({
+                  selectedTags: [...this.state.selectedTags, instrument]
+              })
```
Comment (rocambille) [inline front/src/App.js]: make it DRY ;)

(Don't Repeat Yourself)

File: front/src/App.js [hunk 2]
```diff
@@ -28,15 +31,47 @@ class App extends React.Component {
...
+    }
+    render() {
+        return (
+            <div className='App'>
+                {this.state.researchIsVisible ? null : (
```
Comment (rocambille) [inline front/src/App.js]: use a more suited operator ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: front/src/App.js (hunk 2 above)
Comment (Brolyus) [reply front/src/App.js]: resolved with 
{this.state.researchIsVisible || (

File: front/src/App.js (hunk 1 above)
Comment (Brolyus) [reply front/src/App.js]: resolved applying setState to the ternary directly

**Comment by Brolyus** [comment]
Corrected

**Pull request merged**